std/parallelism.d

```diff
@@ -74,21 +74,9 @@ License:    $(WEB boost.org/LICENSE_1_0.txt, Boost License 1.0)
 */
 module std.parallelism;
 
-import core.atomic;
-import core.exception;
-import core.memory;
-import core.sync.condition;
-import core.thread;
-
-import std.algorithm;
-import std.conv;
-import std.exception;
-import std.functional;
-import std.math;
-import std.meta;
-import std.range;
-import std.traits;
-import std.typecons;
+import core.thread : Thread;
+
+import std.traits : ReturnType;
```

Comment:
Comment these symbols out, selective imports are not allowed at module scope in Phobos yet.  See [related article](http://www.schveiguy.com/blog/2016/03/import-changes-in-d-2-071/) and [forum post](http://forum.dlang.org/post/bsaekthwaqrvszexpxns@forum.dlang.org).